clients/nodejs/validator-registration.js

```diff
@@ -0,0 +1,354 @@
+     * (All sizes in bytes)
+     *
+     * <- 1  -><- 11 -> <-    32    -><-      20        ->
+     * | Type | Unused | Signing key | Validator Address |
+     *
+     * <- 1  -><- 6   -> <-           57                ->
+     * | Type | Unused |        Voting key[0]            |
+     *
+     * <- 1  -><- 6   -> <-           57                ->
+     * | Type | Unused |        Voting key[1]            |
+     *
+     * <- 1  -><- 6   -> <-           57                ->
+     * | Type | Unused |        Voting key[2]            |
+     *
+     * <- 1  -><- 6   -> <-           57                ->
+     * | Type | Unused |        Voting key[3]            |
+     *
+     * <- 1  -><- 6   -> <-           57                ->
+     * | Type | Unused |        Voting key[4]            |
+     */
+    get_serialized_data() {
+        const txns_data = [];
+
+        // Serialize first part of the transaction data
+        let data = new Nimiq.SerialBuffer(64);
+        // First byte: type
+        let type = 1;
+        data.writeUint8(type);
+
+        // Unused bytes
+        const unused = new Uint8Array(11);
+        data.write(unused);
+
+        // Signing key and validator address
+        this._signing_key.serialize(data);
+        this._validator_address.serialize(data);
+
+        // We are done with the first part of the transaction data
+        txns_data.push(data);
+
+        // Extract the voting key
+        const voting_key = Nimiq.BufferUtils.fromHex(this._voting_key.toHex());
+
+        if (voting_key.length != 285) {
+            console.error('Invalid voting key');
+            process.exit(1);
+        }
+
+        // Create the next transactions
+        const vk_unused = new Uint8Array(6);
+
+        for (let i = 0; i < 5; i++) {
+            data = new Nimiq.SerialBuffer(64);
+            type += 1;
+            data.writeUint8(type);
+            data.write(vk_unused);
+            // We read the voting key in 57 bytes chunks
+            data.write(voting_key.read(57));
+            txns_data.push(data);
+        }
+
+        return txns_data;
+    }
+}
+
+function help() {
+    console.log(`Nimiq NodeJS tool to register a new validator
+
+Usage:
+    node validator.js [options]
+
+Description:
+    If no arguments are provided all validator keys are generated and stored in a JSONC file
+    Otherwise, the user must specify the path to the JSONC file with the validator configuration:
+    The Validator Account, the Signing Key and the Voting Key.
+    The nimiq core-web package is necessary to execute this tool, this can be installed via:
+    $npm install @nimiq/core-web@next
+
+Options:
+    --help             Display this help page
+    --validator        Path to the validator specification file (JSONC)
+    --network          The network that should be used to register a validator. 
+                       If this argument is not provided, we connect to the main network by default
```

Comment:
```suggestion
                       If this argument is not provided, we connect to the test network by default
```

As stated on line 121

Reply:
I would either make `test` default or make the parameter not have a default value (mandatory).